macro/o2sim.C

```diff
@@ -84,6 +84,9 @@ FairRunSim* o2sim_init(bool asservice)
   run->SetName(confref.getMCEngine().c_str()); // Transport engine
   run->SetIsMT(confref.getIsMT());             // MT mode
 
+  /** set external decayer **/
+  run->SetPythiaDecayer(TString(confref.getDecayConfig()));
```

Comment:
so by default this function will pass an empty string to SetPythiaDecayer().
What is this default behaviour? No decay or some standard configuration for decay?

Reply:
if the string is empty `FairRoot` will load the default configuration from
```
$VMCWORKDIR/Detectors/gconfig/DecayConfig.C
``` 
which at `o2` installation is taken from 
```
Detectors/gconfig/DecayConfig.C
```

Let's say that this file should contain the default configuration.
Honestly, I am not completely happy about the way this is done.
But this is the minimal change to have the feature working + have a way to change the configuration at runtime.

I plan to look into this a bit more and in case come with a different approach.

Reply:
Ok thanks.